MdePkg/Library/BaseFdtLib/BaseFdtLib.inf

```diff
@@ -57,6 +57,9 @@
   BaseMemoryLib
 
 [BuildOptions]
-  MSFT:*_*_IA32_CC_FLAGS = /wd4146 /wd4245
-  MSFT:*_*_X64_CC_FLAGS  = /wd4146 /wd4244 /wd4245 /wd4267
+# warning C4706: assignment within conditional expression
+# if ((err = fdt_splice_(fdt, p, oldlen, newlen)))
+# in BaseFdtLib\libfdt\libfdt\fdt_rw.c (wait for sub module update to remove this)

```

Comment:
Same change here has been approved by Liming in another PR:
https://github.com/tianocore/edk2/pull/6094#issuecomment-2309292299

Reply:
just FYI